x-pack/auditbeat/auditbeat.reference.yml

```diff
@@ -317,6 +317,7 @@ auditbeat.modules:
 #    match_source_index: 4
 #    match_short_id: false
 #    cleanup_timeout: 60
+#    labels.dedot: false
```

Comment:
Where is that coming from?

Reply:
Sorry that's from not pulling the latest code... I just did a rebase and it's gone.